randovania/games/samus_returns/logic_database/Area 4 Crystal Mines.txt

```diff
@@ -707,7 +707,7 @@ Extra - asset_id: collision_camera_010
 
 > Door to Tsumuri Tunnel; Heals? False
   * Layers: default
-  * Access Locked to Tsumuri Tunnel/Door to Mines Entrance
+  * Access Locked to Tsumuri Tunnel/Door to Mines Entrance; Excluded from Dock Lock Rando
```

Comment:
might wanna put into the node info as to why this is the case

Reply:
Done